wallet/src/de/schildbach/wallet/ui/dashpay/PlatformRepo.kt

```diff
@@ -145,12 +144,6 @@ class PlatformRepo @Inject constructor(
     }
 
     suspend fun init() {
-        if (!platformSDKLoaded) {
-            // load the dash-sdk library
-            System.loadLibrary("sdklib")
-            platformSDKLoaded = true
-        }
-
```

Comment:
the `Platform` class which is used in `PlatformServiceImplementation` also loads the `sdklib`.